ArduCopter/version.h

```diff
@@ -1,12 +1,15 @@
 #pragma once
 
-#include "ap_version.h"
+#include "AP_Common/AP_FWVersion.h"
 
 #define THISFIRMWARE "APM:Copter V3.6-dev"
+
+// the following line is parsed by the autotest scripts
 #define FIRMWARE_VERSION 3,6,0,FIRMWARE_VERSION_TYPE_DEV
```

Comment:
> I do want to fix that... somehow... possibly compiling a "version" program.

You don't actually need a version program. You only need a version.o  (and version.h). The intention was to leave it as isolated as possible, so when you commit a change to git it only needs to rebuild this single file since the commit hash changed.

I agree about the `THISFIRMWARE` statement.